nms/app/packages/magmalte/app/views/equipment/FEGGatewayDetailSubscribers.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+import type {FEGGatewayDetailType} from './FEGGatewayDetailMain';
+import type {subscriber} from '@fbcnms/magma-api';
+
+import ActionTable from '../../components/ActionTable';
+import FEGSubscriberContext from '../../components/context/FEGSubscriberContext';
+import Link from '@material-ui/core/Link';
+import LoadingFiller from '@fbcnms/ui/components/LoadingFiller';
+import React from 'react';
+import nullthrows from '@fbcnms/util/nullthrows';
+
+import {FetchSubscribers} from '../../state/lte/SubscriberState';
+import {
+  REFRESH_INTERVAL,
+  RefreshTypeEnum,
+  useRefreshingContext,
+} from '../../components/context/RefreshContext';
+import {useEffect, useState} from 'react';
+import {useRouter} from '@fbcnms/ui/hooks';
+
+type SubscriberRowType = {
+  name: string,
+  id: string,
+  service: string,
+};
+
+/**
+ * Returns a table of subscribers serviced by the federation gateway.
+ *
+ * @param {FEGGatewayDetailType} props An object containing the federation gateway and a boolean
+ *  telling to refresh or not
+ */
+export default function GatewayDetailSubscribers(props: FEGGatewayDetailType) {
```

Comment:
Don't just add an annotation describing what fields the argument object contains - people can easily see that by checking the type definition. It's better to just leave the annotation bare-bones if there's nothing helpful to put.